index.html

```diff
@@ -1,35 +1,99 @@
-			</div>
-			<ul class="nav-bar-main-menu">
-				
-			</ul>
-		</div>
-	</div>
-	</nav>
-	<!------------------------ NAV BAR END ------------------------>
+    <!------------------------ NAV BAR START ------------------------>
+    <nav>
+        <div class="nav-bar-wrapper">
+            <!-- TODO: nav bar -->
+            <!-- TODO: Label different classes as active for navi bar : https://stackoverflow.com/questions/26819675/navbar-highlight-for-current-page"-->
+            <div>
+                <div class="nav-bar-logo">
+                    <!-- <a href="home"><img src="images/new_oscillate_logo.png" alt="Oscillate"></a> -->
+                </div>
+                <div class="nav-bar-main-menu">
+                    <div class="dropdown">
+                        <button class="dropbtn">TEAM</button>
+                        <div class="dropdown-content team-dropdown-content">
+                            <a href="http://2018.igem.org/Team:Cornell/Team">BIOS</a>
+                            <a href="http://2018.igem.org/Team:Cornell/Sponsors">SPONSORS</a>
+                            <a href="http://2018.igem.org/Team:Cornell/Attributions">ATTRIBUTIONS</a>
+                        </div>
+                    </div>
+                    <div class="dropdown">
+                        <button class="dropbtn">OUTREACH</button>
+                        <div class="dropdown-content outreach-dropdown-content">
+                            <a href="#">COLLABORATIONS</a>
+                            <a href="#">PUBLIC ENGAGEMENT</a>
+                        </div>
+                    </div>
+                    <div class="dropdown">
+                        <button class="dropbtn">HUMAN-CENTERED DESIGN</button>
+                        <div class="dropdown-content human-dropdown-content">
+                            <a href="#">PRACTICES</a>
+                            <a href="#">POLICIES</a>
+                            <a href="http://2018.igem.org/Team:Cornell/Entrepreneurship">ENTREPRENEURSHIP</a>
+                        </div>
+                    </div>
+                    <div class="dropdown">
+                        <button class="dropbtn">MODELING</button>
+                        <div class="dropdown-content modeling-dropdown-content">
+                            <a href="http://2018.igem.org/Team:Cornell/Model">MODEL</a>
+                            <a href="http://2018.igem.org/Team:Cornell/Software">SOFTWARE</a>
+                            <a href="http://2018.igem.org/Team:Cornell/Design">DESIGN</a>
+                        </div>
+                    </div>
+                    <div class="dropdown">
+                        <button class="dropbtn">TOOLKIT</button>
+                        <div class="dropdown-content toolkit-dropdown-content">
+                            <ul>
+                                <div class="nav-first-col">
+                                    <li><a href="#"><b>WET LAB</b></a>
```

Comment:
Can you make it so that hovering over WET LAB doesn't change color? So there should be no white button when you hover over WET LAB.